doc/lua_api.txt

```diff
@@ -2682,6 +2697,14 @@ and `minetest.auth_reload` call the authetification handler.
     * Convert a vector into a yaw (angle)
 * `minetest.yaw_to_dir(yaw)`
     * Convert yaw (angle) to a vector
+* `minetest.is_colored_paramtype(ptype)`
+    * returns if the given `paramtype2` is `color`,
```

Comment:
<pre>
    * returns if the given `paramtype2` contains color information.
      namely: `color`, `colorwallmounted` or `colorfacedir`
</pre>